src/components/BookMark/BookMark.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class="BookMark">{{ label }}</div>
+</template>
+
+<script lang="ts">
+import './styles.scss';
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'BookMark',
+  props: {
+    label: {
+      type: String,
+      default: '',
+    },
+  },
+});
+</script>
```

Comment:
same here

Reply:
Can you check if on your code editor it exists? on my code editor it does :

![image](https://user-images.githubusercontent.com/66505715/173159492-d624aa45-9755-4e0e-9633-fe35a76236ef.png)

But github is not showing it